webroot/src/store/pagination/pagination.state.ts

```diff
@@ -20,6 +20,12 @@ export const paginationTemplate = {
     size: DEFAULT_PAGE_SIZE,
 };
 
+export interface PageChangeConfig {
+    firstIndex: number;
+    lastIndexExclusive: number;
+    prevNext: number;
```

Comment:
Don't hate me too much...but mind adding a brief comment explaining this? Or perhaps this deserves a more descriptive name? I'm not sure, but bottom line is that I read this as "This is an interface that represents the current state of pagination, where `firstIndex` and `lastIndexExclusive` represent the range of items in a list that are displayed."

`prevNext` is where I get a little confused and am unable to accurately describe. Is `prevLastIndexExclusive` more accurate?

Reply:
I added a comment for `prevNext`. It's something needed to support the backend's current approach to paging.